CRM/Contribute/Form/Contribution/Main.php

```diff
@@ -1338,9 +1338,10 @@ public function assignFormVariablesByContributionID(): void {
     if (empty($this->getExistingContributionID())) {
       return;
     }
-    // @todo - all this stuff is likely obsolete.
-    if ($taxAmount = CRM_Core_DAO::getFieldValue('CRM_Contribute_DAO_Contribution', $this->_ccid, 'tax_amount')) {
-      $this->assign('taxAmount', $taxAmount);
+
```

Comment:
`$this-assign('taxAmount',$this-getContributionValue('tax_amount');
 $this->assign('taxTerm', Civi::settings()->get('tax_term'));